test.c

```diff
@@ -0,0 +1,18 @@
+#include <stdlib.h>
```

Comment:
This file should not be included in this PR / or the test should be moved to appropriate path and a testcase should be created